index.js

```
@@ -1520,6 +1579,260 @@
+
+app.post("/admin/approve_message", ensureAdmin, async (req, res) => {
+  try {
+    const { guildId } = req.body;
+    const partner = await Partner.findOne({ guildId });
+    if (partner && partner.pendingMessage) {
+      await Partner.findOneAndUpdate(
+        { guildId },
+        { partnerMessage: partner.pendingMessage, messagePending: false, $unset: { pendingMessage: "" } }
+      );
+      // Notify user if possible
+      if (partner.pendingMessageUserId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)